servers-management/dell/ansible/inventory.yaml

```diff
@@ -0,0 +1,19 @@
+idrac:
+  hosts:
+      openshift-qe-047-drac.mgmt.arm.eng.rdu2.redhat.com:
```

Comment:
why is this one uncommented and the others are not?

Reply:
enabled all hosts in rdu2